cmd/buildah/commit.go

```diff
@@ -38,16 +25,17 @@ var (
 
 func commitCmd(c *cli.Context) error {
 	args := c.Args()
-	name := ""
-	if c.IsSet("name") {
-		name = c.String("name")
+	if len(args) == 0 {
+		return fmt.Errorf("container ID must be specified")
 	}
-	root := c.String("root")
-
-	output := ""
-	if c.IsSet("output") {
-		output = c.String("output")
+	name := args[0]
+	args = args.Tail()
+	if len(args) == 0 {
+		return fmt.Errorf("an image name must be specified")
```

Comment:
I don't think we can ever get into this if statement.  Plus we seem to be setting the name and image to the same value?  Is that correct?